class/deeplink-schema-match.ts

```
@@ -393,59 +61,48 @@
-        if (parsedBitcoinUri.options.amount) {
-          amount = Number(parsedBitcoinUri.options.amount);
-        }
-        if (parsedBitcoinUri.options.label) {
-          memo = parsedBitcoinUri.options.label;
-        }
-        if (parsedBitcoinUri.options.pj) {
-          payjoinUrl = parsedBitcoinUri.options.pj;
-        }
-      }
-    } catch (_) {}
+      const parsedUri = this.bip21decode(uri);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)